index.html

```diff
@@ -72,8 +107,42 @@ <h2 class="card-head">Tonic</h2>
         <li>css</li>
         <li>javaScript</li>
       </ul>
-      <button type="submit" class="btn">See Project</button>
+      <button type="submit" class="btn" onclick="openPopup1()">See Project</button>
+      <!-- START POPUP-->
+      <div class="popup" id="popup1">
+        <div class="popup-content">
+            <div class="pop-head">
+          <h2 class="card-head">Tonic</h2>
+          <a href="javascript:void(0)" class="popup-close" onclick="closePopup1()">&times;</a>
+        </div>
+          <ul class="list">
+        <li class="brand">CANOPY</li>
+        <li><img src="img/Counter.png" alt="Counter">Back End Dev</li>
+        <li><img src="img/Counter.png" alt="Counter">2015</li>
+      </ul>
+        <div class="card-img">
+      <img src="img/Snapshoot(1).png" alt="image">
+    </div>
+    <div class="popup-body">
+      <p class="cards-para">A daily selection of privately personalized reads; no accounts or sign-ups required.Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.</p>
+      <div class="buttons">
+      <ul class="languages">
+        <li>html</li>
+        <li>css</li>
+        <li>javaScript</li>
+      </ul>
+      <div class="popup-b">
+        <button type="submit" class="btn btn-pop" onclick="openPopup()">
+          See Resources <img src="img/Icone.png" alt="Social"></button>
+        <button type="submit" class="btn btn-pop" onclick="openPopup()">
+          See Project <img src="img/Vectors.png" alt="Social"></button>
+        </div>
+      </div>
     </div>
+   </div>
```

Comment:
Mobile Popup: 

- [ ] Kindly make sure to follow the spacing for the list displayed before the image. Also, make sure to follow the color as per the Figma design.
- [ ] Kindly make sure to follow the same height and width for the popup as per the Figma design.
- [ ] Kindly make sure to follow the button text and size as per the Figma design. Also, make sure to align the buttons in the center.
- [ ] Kindly make sure to add a blurred background when the popup is open.

![2022-05-12 (8)](https://user-images.githubusercontent.com/70269507/168032107-5b5d53fb-2614-440f-bdda-fb2f7b8712eb.png)